cmd/root_test.go

```diff
@@ -1,108 +1,85 @@
 package cmd_test
 
 import (
-	"regexp"
 	"testing"
 
 	"github.com/pingidentity/pingctl/internal/customtypes"
-	"github.com/pingidentity/pingctl/internal/testutils"
+	"github.com/pingidentity/pingctl/internal/testutils/testutils_command"
+	"github.com/pingidentity/pingctl/internal/testutils/testutils_helpers"
 )
 
 // Test Root Command Executes without issue
 func TestRootCmd_Execute(t *testing.T) {
-	err := testutils.ExecutePingctl()
-	if err != nil {
-		t.Errorf("Error executing root command: %s", err.Error())
-	}
+	expectedErrorPattern := "" //No error expected
```

Comment:
Could we just pass in nil directly to the function?

Reply:
Good idea, updated!